src/utils/index.js

```diff
@@ -201,6 +201,8 @@ export const copyToBin = libDir => {
 
     if (fs.existsSync(flywayDir)) {
       rimraf.sync(path.join(__dirname, "../../", "bin"));
+
+      fs.removeSync(path.join(flywayDir, "jre", "lib", "amd64", "server", "libjsig.so")) // Broken link, we need to delete it to avoid the copy to fail
```

Comment:
if the `amd64` dir does not exist i think this will also fail. there should be a check that it exists first

Reply:
You're right. I've just added a check before remove.